changelog.json

```diff
@@ -1,5 +1,6 @@
 {
   "releases": {
+    "2.3.2-beta1": [],
```

Comment:
We use `-beta0` to indicate there are no new changes on top of what's currently on production. I'm using `-beta1` here because technically we have a change (#8875), but it's not user-facing so there is no changelog entry.